docs/resources/task.md

```diff
@@ -112,15 +103,13 @@ supported by the underlying cloud provider.
 
 ### Generic
 
-The Iterative Provider offers some common machine images which are roughly the same
-for all supported clouds.
+The Iterative Provider offers some common machine images which are roughly the same for all supported clouds.
 
-- `ubuntu` - Official Ubuntu LTS image, currently 20.04.
+- `ubuntu` - [Latest](https://hub.docker.com/_/ubuntu) official [Ubuntu LTS](https://wiki.ubuntu.com/LTS) image.
```

Comment:
```suggestion
- `ubuntu` - [Latest](https://hub.docker.com/_/ubuntu) official [Ubuntu LTS](https://wiki.ubuntu.com/LTS) image.
```

Virtual machine images are hardcoded: a new Ubuntu LTS release does **not** imply an immediate bump in our default images.

* Cloud images usually lag behind official LTS releases.
* We aren't going to upgrade them until they are supported by every provider.

Reply:
ah so didn't you mean to suggest:

```suggestion
- `ubuntu` - Official [Ubuntu LTS](https://wiki.ubuntu.com/LTS) image (currently 20.04).
```

Reply:
🤦🏼 Sorry! The suggestion was broken.